integration-tests/e2e/list-page.cy.ts

```diff
@@ -40,6 +40,28 @@ describe(`${PLUGIN_NAME} list page test`, () => {
     cy.get(".pf-c-alert.pf-m-inline.pf-m-danger").should("not.exist");
     cy.byLegacyTestID("resource-title").should("contain", "my-new-cron-object");
 
+    cy.log("Updating the labels of the new CronTab from the list page");
+    cy.visit(namespacedListPageURL);
+    listPage.rows.shouldBeLoaded();
+    listPage.rows.shouldExist("my-new-cron-object");
+    listPage.rows.clickKebabAction("my-new-cron-object", "Edit Labels");
+    modal.shouldBeOpened();
+    cy.byLegacyTestID("modal-title").should("have.text", "Edit labels");
+    modal.submitShouldBeEnabled();
+    cy.byTestID("tags-input").type("key1=value1{enter}");
+    cy.get(".tag-item-content .pf-c-label__content").should(
```

Comment:
I would avoid accessing elements like this. Lets ratherre/use helpers that we already have in console repo. I've reused them in https://github.com/openshift/crontab-plugin/pull/8/files#diff-b1dce0141c3765e3e888abdc79f28116a9e74498207b3d6e1aa74233c794686b